src/fragments/lib/pubsub/js/subunsub.mdx

```diff
@@ -56,4 +56,25 @@ const sub1 = PubSub.subscribe('myTopicA').subscribe({
 
 sub1.unsubscribe();
 // You will no longer get messages for 'myTopicA'
-```
\ No newline at end of file
+```
+
+## Subscription connection status updates
+
+Now that your application is setup and using pubsub subscriptions, you may want to know when the subscription is finally up and running, or reflect to your users when the subscription isn't healthy. You can monitor the connection state for changes via Hub.
```

Comment:
I prefer to be more concrete like: you may want to know when the subscription is finally **established**, or reflect to ...

What do you think @stocaaro ?

Reply:
Updated